internal/cloud/backend_plan.go

```diff
@@ -468,7 +468,7 @@ func (b *Cloud) renderPlanLogs(ctx context.Context, op *backend.Operation, run *
 					// print the line. This maintains backwards compatibility for
 					// users who do not wish to enable structured output in their
 					// workspace.
-					b.CLI.Output(string(line))
+					b.CLI.Output(b.Colorize().Color(string(line)))
```

Comment:
Can you describe what's going on here? It seems like we wouldn't have/want json log lines to contain terminal color escapes.

Reply:
If we cannot parse the JSON, do we not want to colorize what is printed to the CLI?

Reply:
No relevant bug here. Just updating how it printed. Will revert.